presentation/src/main/java/com/open/presentation/ui/components/ErrorCard.kt

```diff
@@ -0,0 +1,89 @@
+package com.composeweatherapp.presentation.component
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.Card
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextAlign
+
+
+@Composable
+fun ErrorCard(
+    modifier: Modifier = Modifier,
+    errorTitle: String,
+    errorDescription: String,
+    errorButtonText: String,
+    onClick: () -> Unit,
+    cardModifier: Modifier,
```

Comment:
How is `modifier` different than `cardModifier` if the composable's name is Error`Card`? Do you see what I'm trying to say here?

Reply:
understood